woocommerce/woocommerce
# Adding Venezuelan states

### All Submissions:






### Changes proposed in this Pull Request:


Added Venezuelan states to WooCommerce localization settings

### How to test the changes in this Pull Request:

1. Switch to this pr branch
2. Go to checkout page, check that Venezuelan states are an option. Can also be seen in setup wizard or within WooCommerce settings > general > country/state.

### Other information:




### Changelog entry

> Localization - Added Venezuelan states.
